be/src/runtime/exec_env.cpp

```diff
@@ -288,13 +288,11 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
                                         std::make_unique<workgroup::WorkGroupScanTaskQueue>(
                                                 workgroup::WorkGroupScanTaskQueue::SchedEntityType::OLAP));
     _scan_executor_with_workgroup->initialize(num_io_threads);
-    // it means acting as compute node while store_path is empty. some threads are not needed for that case.
-    if (!store_paths.empty()) {
-        Status status = _load_path_mgr->init();
-        if (!status.ok()) {
-            LOG(ERROR) << "load path mgr init failed." << status.get_error_msg();
-            exit(-1);
-        }
+
+    Status status = _load_path_mgr->init();
```

Comment:
what will `_load_path_mgr->init()` returns if storage_paths is empty? fail or crash or still ok?